datumaro/components/extractor.py

```diff
@@ -127,7 +131,10 @@ def __iter__(self):
 @attrs
 class Label(Annotation):
     _type = AnnotationType.label
-    label = attrib(converter=int)
+    label = attrib(converter=int, default=0)
+    name = attrib(type=str, default=None)
+    color = attrib(default=None, type=str)
```

Comment:
`Label`s can't have colors, they are just classes. If you need a color for a class, use `MaskCategories`. Labels in annotation classes (`Label`, `Bbox` etc) are indexes in the `LabelCategories`, which must be defined for a dataset, if it has labels. Other `Categories` classes (`MaskCategories`, `KeypointCategories`) also refer to `LabelCategories` by indices.

Reply:
Resolved